pkg/noun/events.c

```diff
@@ -1379,7 +1380,9 @@ u3e_save(u3_post low_p, u3_post hig_p)
     }
   }
 
-  u3e_backup(c3n);
+  c3_c bhk_c[8193];
+  snprintf(bhk_c, sizeof(bhk_c), "%s/.urb/bhk", u3P.dir_c);
+  u3e_backup(bhk_c, c3n);
```

Comment:
This heuristic backup should be removed, the epoch system itself is a much better solution.